test/spec/geo/geocoder.spec.js

```diff
@@ -44,6 +44,51 @@ describe('Geocoder', function() {
     });
   });
 
+describe('MAPZEN', function() {
+
+    it("don't remove the spaces in the user-submitted addresses [MAPZEN]", function(done) {
+      var regexp = new RegExp(/http:\/\/search.mapzen.com\/v1\/search\?text\=bn20%208qt/);
+     
+
+      cdb.geo.geocoder.MAPZEN.geocode('bn20 8qt', function(d) {
+        expect(this.url).toMatch(regexp);
+        done();
+      });
+    });
+
+    it("we should get a direction that exists using MAPZEN", function(done) {
+      var data;
+      cdb.geo.geocoder.MAPZEN.geocode('Madrid, Spain', function(d) {
```

Comment:
@oriolbx please try to fix these linter errors. In this case for example, using single quotes instead of double quotes would do it.